pkgs/servers/invidious/default.nix

```diff
@@ -8,17 +8,16 @@ let
   #  * shards.nix (by running `crystal2nix` in invidious’ source tree)
   #  * If the lsquic.cr dependency changed: lsquic in lsquic.nix (version, sha256)
   #  * If the lsquic version changed: boringssl' in lsquic.nix (version, sha256)
```

Comment:
I think this should be changed to reflect that the versions/hashes are now stored in a separate file (and no longer in `lsquic.nix` or `default.nix`).